README.md

```diff
@@ -3,10 +3,19 @@ Pancake
 
 > Pancake is a tool to make working with npm on the front end easy and sweet.
 
+![The Pancake tool](https://raw.githubusercontent.com/govau/pancake/master/assets/pancake.png)
+
+[Npm wrote about](http://blog.npmjs.org/post/101775448305/npm-and-front-end-packaging) the challenges frontend developers face when trying to use npm. Pancake
+is addressing those by embracing the idea of small individually versioned independent modules. Interdependencies is what npm does really well and Pancake
+will help you keep them flat and error out on conflicts. [Read more about our solution](https://medium.com/dailyjs/npm-and-the-front-end-950c79fc22ce)
+
+Pancake will check your peerDependencies for conflicts and comes with plugins to compile the contents of your modules for you and lists all available modules for you to select and install.
```

Comment:
Should "peerDependencepies" be camel case? Feels odd that its like that in a sentence